spec/requests/graphql/queries/events/get_events_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe Types::QueryType do
+  describe 'display events' do
+    it 'can query all eventss' do
+      create_list(:event, 5)
+
+      post graphql_path, params: { query: query }
+      result = JSON.parse(response.body)
+
+      # I think this test may be able to be moved into Leah's user spec, as I've just added Events to be queried on a user.
```

Comment:
See here, I can clean this up once we merge

Reply:
Referencing my comment above on not having a getEvents query.. you could just move this test in to the getUser spec like you mentioned in the comment!